clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/StoreIngestionTask.java

```diff
@@ -176,6 +176,9 @@ public abstract class StoreIngestionTask implements Runnable, Closeable {
   private static final ExecutorService SHUTDOWN_EXECUTOR_FOR_DVC =
       Executors.newFixedThreadPool(Runtime.getRuntime().availableProcessors() * 2);
 
+  protected static final ChunkedValueManifestSerializer CHUNKED_VALUE_MANIFEST_SERIALIZER =
+      new ChunkedValueManifestSerializer(true);
```

Comment:
Yeah. There would be perf concerns too, I believe

Reply:
I worry about that too, which is why it wasn't like this in the original PR.

I noticed that `ChunkedValueManifestSerializer#deserialize()` doesn't seem to be writing anything to the instance variables, so I thought it might be safe to initialize the instance once and reuse it many times.

I saw another class that had this serializer as `static` (`LeaderProducerCallback`, albeit I think it's a single-use class?) and I wanted to propose this. Just not sure if it's safe to do so in Java?

Reply:
I feel that there could still might be code path to change its internal state, e.g. `cacheDatumReader` will modify `protocolVersionToReader`. If we would like to make this change, maybe in a separate PR so that it would catch people's attention.

Reply:
Oh I see, I'll revert it then. I just wanted to propose it in a small PR so it was easier to see.